oiosaml/src/main/java/dk/gov/oio/saml/config/Configuration.java

```diff
@@ -9,7 +9,13 @@
     // SP configuration
     private String spEntityID; // This SP's EntityID
     private String baseUrl; // The URL endpoint that the DispatcherServlet is working on
-    private boolean validationEnabled = true;
+	private String servletRoutingPathPrefix; // The endpoint prefix that the DispatcherServlet is working on
```

Comment:
There is some issues with tabs versus spaces. Could reformat to 4 spaces as the rest?

Reply:
The prefix and suffixes must be optional configuration parameters

Reply:
done